haproxy/tests/test_unit.py

```diff
@@ -202,13 +203,13 @@ def test_count_hosts_statuses(aggregator, haproxy_mock):
     haproxy_check._process_data(data, True, False, collect_status_metrics=True,
                                 collect_status_metrics_by_host=True)
     expected_hosts_statuses = defaultdict(int)
-    expected_hosts_statuses[('b', 'FRONTEND', 'open')] = 1
-    expected_hosts_statuses[('a', 'FRONTEND', 'open')] = 1
-    expected_hosts_statuses[('b', 'i-1', 'up')] = 1
-    expected_hosts_statuses[('b', 'i-2', 'up')] = 1
-    expected_hosts_statuses[('b', 'i-3', 'up')] = 1
-    expected_hosts_statuses[('b', 'i-4', 'down')] = 1
-    expected_hosts_statuses[('b', 'i-5', 'maint')] = 1
+    expected_hosts_statuses[('b', 'FRONTEND', 'FRONTEND', 'open')] = 1
+    expected_hosts_statuses[('a', 'FRONTEND', 'FRONTEND', 'open')] = 1
```

Comment:
duplication of `FRONTED` on purpose ?

Reply:
I don't know details but that's what the function does indeed